### PR TITLE
{bio}[gompi/2019b] MMseqs2 v10-6d92c

### PR DESCRIPTION
**Pull request by lexming**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
login2.cerberus.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/36113d3773a5feeb6a9b03ada484e292 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
generoso - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v4 @ 2.00GHz, Python 3.6.8
See https://gist.github.com/461e51d2cc12aa0da9755cb00e234de9 for a full test report.

File: easybuild/easyconfigs/m/MMseqs2/MMseqs2-10-6d92c-gompi-2019b.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+easyblock = 'CMakeMake'
```
Comment (verdurin) [inline easybuild/easyconfigs/m/MMseqs2/MMseqs2-10-6d92c-gompi-2019b.eb]: @lexming as with the other one, please reinstate the previous attribution

Comment (lexming) [reply easybuild/easyconfigs/m/MMseqs2/MMseqs2-10-6d92c-gompi-2019b.eb]: fixed :+1:

Comment (verdurin) [reply easybuild/easyconfigs/m/MMseqs2/MMseqs2-10-6d92c-gompi-2019b.eb]: Thanks.

**Comment by verdurin** [comment]
Going in, thanks @lexming!

**Pull request merged**